test/mean-module.test.js

```diff
@@ -0,0 +1,65 @@
+'use strict';
+
+var path = require('path'),
+  helpers = require('yeoman-generator').test,
+  assert = require('yeoman-generator').assert,
+  temp = require('temp').track();
+
+describe('Module Sub Generators Tests', function () {
+  this.timeout(0);
+  /**
+   * Setup the temp directory
+   */
+  before(function (done) {
+    helpers.testDirectory(path.join(__dirname, 'temp'), done);
+  });
+
+  /**
+   * Clean up temp directory
+   */
+  after(function () {
+    temp.cleanup();
+  });
+
+  describe('Generate an express controller through the sub-generator', function () {
+    beforeEach(function (done) {
+      helpers.run(path.join(__dirname, '../mean-module'))
+        .withOptions({
+          'skip-install': true
+        })
+        .withArguments(['foo'])
```

Comment:
Just not fixing what's not broken - 0.3.0 subgenerators worked this way, so people got already used to them, and I left that logics be.
I can change it to a prompt, actually, but either way seems ok to me.